miso-web/src/main/webapp/scripts/list_dilution.js

```diff
@@ -83,11 +83,14 @@ ListTarget.dilution = {
   createColumns: function(config, projectId) {
     return [{
       "sTitle": "Name",
-      "mData": "id",
+      "mData": "name",
```

Comment:
Can this be changed back to `id`? Using `name` causes the column to be sorted alphabetically (1,10,100,1000,10000,10001, etc.)

Reply:
Fixed. Added comment to explain